Engine/Cli/CliActionExecutor.cs

```diff
@@ -269,26 +269,34 @@ string getVersion()
 
                     return TypeData.FromType(typeof(CliActionExecutor)).Assembly.SemanticVersion.ToString(); // OpenTAP is not installed. lets just return this. 
                 }
+                string tapCommand = OperatingSystem.Current == OperatingSystem.Windows ? "tap.exe" : "tap";
 
-                Console.WriteLine("OpenTAP Command Line Interface ({0})", getVersion());
-                Console.WriteLine("Usage: tap <command> [<subcommand(s)>] [<args>]\n");
+                log.Info("OpenTAP Command Line Interface ({0})", getVersion());
+                if (args.Length == 0)
+                {
+                    log.Info($"\"{tapCommand}\" is not a recognized command.");
```

Comment:
i think we can skip printing this line - if the user just writes "tap.exe" we can assume they just want the help. What do you think?

Reply:
Yeah, this probably makes sense. Change has been made.